CHANGELOG.rst

```diff
@@ -4,6 +4,13 @@ Ansible Collection cloudscale.ch Release Notes
 
 .. contents:: Topics
 
+unreleased
```

Comment:
Changelogs are added using changelog fragments, see https://docs.ansible.com/ansible/latest/community/development_process.html#creating-a-changelog-fragment.

I think in this case, this can be completely left out however. I don't think this even reaches "minor change" level:

> Minor changes to ansible-core, modules, or plugins. This includes new parameters added to modules, or non-breaking behavior changes to existing parameters, such as adding additional values to choices[]. Minor changes are enhancements, not bug fixes. Write in present tense.

Basically you could just leave out the CHANGELOG.rst change.

Reply:
Nice, thanks a lot!